### PR TITLE
feat: Styles CartItem

### PR DESCRIPTION
**Pull request by danzanzini**
## What's the purpose of this pull request?
This PR styles the CartItem following this [jira task](https://vtex-dev.atlassian.net/browse/FSSS-139).

## How does it work? 
The `CartItem` component already existed. I've added some styles to it. A small adjustment at the `QuantitySelector` component was necessary.

## How to test it?
Access the preview. Add some items to the cart and open the cart. For now, it looks like this:
image

Thanks to @renatamottam for the help

### CODE REVIEW

File: src/components/ui/QuantitySelector/QuantitySelector.tsx [hunk 1]
```diff
@@ -7,15 +7,17 @@ import './quantity-selector.scss'
...
 }
 
 export function QuantitySelector({
   max,
   min,
+  initial,
   disabled,
 }: QuantitySelectorProps) {
-  const [quantity, setQuantity] = useState<number>(min)
+  const [quantity, setQuantity] = useState<number>(initial ?? min)
```
Comment (saranicoly) [inline src/components/ui/QuantitySelector/QuantitySelector.tsx:20]: I'm curious about this `??` operator

**Review by saranicoly — APPROVED** [review]
LGTM. Good job!

File: src/components/ui/QuantitySelector/QuantitySelector.tsx (hunk 1 above)
Comment (israelswf) [reply src/components/ui/QuantitySelector/QuantitySelector.tsx:20]: @saranicoly, it is the [Nulllish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).

There is also [this good chapter](https://www.executeprogram.com/courses/everyday-typescript/lessons/nullish-coalescing) in the Execute Program: Everyday TypeScript.

**Review by filipewl — APPROVED** [review]
Looking good!

**Comment by danzanzini** [comment]
@israelswf I've just added the QuantitySelector but I haven't added any behavior to it. If you change the value, will not affect the cart.

**Pull request merged**